### PR TITLE
[Feature] Filling in personal data in User profile

### PR DESCRIPTION
**Pull request by gogochikago**
**Link to the ticket**
https://trello.com/c/UarxLnED/11-feature-filling-in-personal-data-in-user-profile
**What it does**
Added creating, editing and deleting a profile. 
**DB changes**
Created table _profiles_.
**UI changes**
Created views for ProfilesController.

### CODE REVIEW

**Comment by gogochikago** [comment]
[[Feature] Filling in personal data in User profile](https://trello.com/c/UarxLnED/11-feature-filling-in-personal-data-in-user-profile)

File: spec/controllers/profiles_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,138 @@
+require 'rails_helper'
+
+RSpec.describe ProfilesController, type: :controller do
+  let!(:profile) { Profile.create(valid_params)}
+  let!(:valid_params) do
```
Comment (labs-ruby) [inline spec/controllers/profiles_controller_spec.rb]: No need to use `let!` here, `valid_params` just returns a hash

File: spec/controllers/profiles_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    end
+
+    it 'rednder profiles#edit template' do
+      get :edit, params: { id: profile.id }
+      expect(response).to render_template(:edit)
+    end
+  end
+
+  describe 'PUT #update' do
+    let!(:valid_attribute) do
```
Comment (labs-ruby) [inline spec/controllers/profiles_controller_spec.rb]: The same, + fix indentation

File: spec/helpers/profiles_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (labs-ruby) [inline spec/helpers/profiles_helper_spec.rb]: Delete empty files

File: spec/views/profiles/new.html.slim_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (labs-ruby) [inline spec/views/profiles/new.html.slim_spec.rb]: Delete empty file

**Pull request merged**